### PR TITLE
refactor(android): All android code is removed

### PR DESCRIPTION
**Pull request by Diadlo**
#3168

>  Files changed 37 

Should I divide it in few commits to easy reading?

### CODE REVIEW

File: qtox.pro [hunk 1]
```diff
@@ -272,135 +230,133 @@ unix:!macx:!android {
...
-        src/widget/form/tabcompleter.h \
-        src/net/autoupdate.h \
-        src/widget/tool/callconfirmwidget.h \
-        src/widget/systemtrayicon.h \
-        src/widget/qrwidget.h \
-        src/widget/systemtrayicon_private.h \
-        src/widget/loginscreen.h \
-        src/ipc.h
+RESOURCES += res.qrc \
+    smileys/smileys.qrc
```
Comment (zetok) [inline qtox.pro:262]: Could you move `RESOURCES` after the platform-specific stuff below?

File: qtox.pro [hunk 2]
```diff
@@ -272,135 +230,133 @@ unix:!macx:!android {
...
+
+HEADERS  += \
```
Comment (zetok) [inline qtox.pro:264]: Either add a comment that here's the stuff that is not portable/useful on android, or merge it with `HEADERS` below.

File: qtox.pro [hunk 3]
```diff
@@ -272,135 +230,133 @@ unix:!macx:!android {
...
-        src/chatlog/content/timestamp.cpp \
-        src/chatlog/documentcache.cpp \
-        src/chatlog/pixmapcache.cpp \
-        src/persistence/offlinemsgengine.cpp \
-        src/widget/qrwidget.cpp \
-        src/widget/genericchatroomwidget.cpp \
-        src/widget/loginscreen.cpp \
-        src/ipc.cpp
-}
+SOURCES += \
```
Comment (zetok) [inline qtox.pro:383]: Either add a comment that here's the stuff that is not portable/useful on android, or merge it with `SOURCES` below.

File: src/widget/gui.cpp [hunk 4]
```diff
@@ -29,22 +29,14 @@
 #include <QMessageBox>
 #include <QPushButton>
 #include <QThread>
-
-#ifdef Q_OS_ANDROID
-#include "androidgui.h"
-#else
 #include "widget.h"
```
Comment (zetok) [inline src/widget/gui.cpp]: Shouldn't that be moved up?

**Comment by zetok** [comment]
Could you also add `Closes #3168` in commit message body?

**Comment by Diadlo** [comment]
@zetok All was fixed

**Comment by antis81** [comment]
LGTM :+1:

**Comment by c33s** [comment]
so andorid support is dropped?
then the information on https://tox.chat/clients.html should be changed. currently it states android support `Operating Systems:    Linux, Windows, OSX, BSD, Android`

**Comment by sudden6** [comment]
Android support never really worked, no idea how it got there. I submitted a PR to the website: https://github.com/Tox/tox.chat/pull/107

**Pull request merged**